src/integration/common/core.canonical.js

```diff
@@ -31,4 +31,13 @@ export default ({ service }) => {
 
     expect(serviceScripts.length).toBe(1);
   });
+
+  // it('Page data used for client side app hydration', () => {
+  //   const simorghDataScriptEl = Array.from(
+  //     document.querySelectorAll('script'),
+  //   ).find((script) => script.textContent.startsWith('window.SIMORGH_DATA'));
+
+  //   expect(simorghDataScriptEl).toBeInTheDocument();
+  //   expect(simorghDataScriptEl).toMatchSnapshot();
+  // });
```

Comment:
Would this work if we uncommented it?

Reply:
yeh but it creates huge ugly snapshots. I'm just going to remove it.